java/pgv-artifacts/pom.xml

```diff
@@ -168,7 +168,7 @@
                 <version>3.3.0</version>
                 <executions>
                     <execution>
-                        <id>osx-x86_64</id>
+                        <id>attach-artifacts</id>
```

Comment:
We don't need separate executions for each architecture - we can just list all artifacts in a single execution.